app/Helpers/render/comment.php

```diff
@@ -187,10 +187,10 @@ class="comment-textarea"
                             id="comment_textarea_$commentId"
                             oninput="autoExpandTextInput(this)"
                         ></textarea>
-                        <button class="comment-submit-button">
+                        <button class="btn comment-submit-button">
                             <img src="$submitImageUrl" alt="Submit">
                         </button>
-                        <span class="comment-loading-indicator">
+                        <span class="btn comment-loading-indicator">
```

Comment:
`btn` style on a span?

Reply:
Huh, I thought it needs that for the loading icon to keep the same padding as the submit button but it turned out that loading indicator isn't even in use anymore. Removed it in https://github.com/RetroAchievements/RAWeb/pull/1800/commits/6ffe06ffd98eae79140f4b86646f465f597ed66e